gwt-eclipse-plugin/Download.html

```diff
@@ -128,17 +117,16 @@ <h2 id="2__Install_from_the_Eclipse_Marketplace">2. Install from the Eclipse Mar
        <p>First goto the Eclipse Market Place. <img src="images/EclipseMarketPlace.png" /></p> 
        <p>Then Search for <code>GWT</code> and install the features. <img src="images/SearchForGWT.png" /></p> 
        <p>Then Choose all the features you want to install. <img src="images/MarketPlaceFeatures.png" /></p> 
-       <p>Don’t forget to become a Star Gazer by clicking on the like button! <img src="images/StarGazer.png" /></p> 
        <h2 id="3__Update_Site">3. Update Site</h2> 
        <p>Use the update site to install the features. </p> 
        <p>First go to Install New Software. <img src="images/InstallNewSoftware.png" /></p> 
        <p>Then add the update site and name it <code>GWT Eclipse Plugin</code>. <img src="images/UpdateSite.png" /></p> 
-       <p>Copy and paste this update repository URL and paste it into the <code>Work with</code> in the Eclipse dialog and click enter. <a href="https://plugins.gwtproject.org/eclipse/gwt-eclipse-plugin/4.1.0/">https://plugins.gwtproject.org/eclipse/gwt-eclipse-plugin/4.1.0/</a></p> 
+       <p>Copy and paste this update repository URL and paste it into the <code>Work with</code> in the Eclipse dialog and click enter. <a href="https://plugins.gwtproject.org/eclipse/site/latest/">https://plugins.gwtproject.org/eclipse/site/latest/</a></p> 
       </div> 
       <div class="tpl-content-lastUpdated">
         Last updated 
-       <div id="tpl-content-lastUpdated-date" itemprop="datePublished" datetime="2024-09-25T12:02Z">
-        September 25, 2024
+       <div id="tpl-content-lastUpdated-date" itemprop="datePublished" datetime="2025-02-25T17:25Z">
+        Februar 25, 2025
```

Comment:
```suggestion
        February 25, 2025
```

Reply:
That's seems to be generated. Maybe it is "Februar" because i'm in Germany?